analogin.py

```diff
@@ -215,4 +236,59 @@ def data_out(self) -> str:
             self.data_string = TCP.format_data('AI/dimensions', shape_str) + \
                 TCP.format_data('AI/data', data_bytes)
                 
-            return self.data_string
\ No newline at end of file
+            return self.data_string
```

Comment:
it isn't used elsewhere at the moment, but it could be a nice thing to have data associated with a device rather than merely spat out to the pxi measurement function in routine looping.